pages/user/index.html

```diff
@@ -45,7 +45,7 @@
               class="rounded-circle d-block mx-auto w-25"
               id="profileImage"
               src="https://images.unsplash.com/photo-1522075469751-3a6694fb2f61?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
-              alt=""
+              alt="This is your profile user avatar for your profile page"
```

Comment:
Maybe it would be better to make it shorter and straight to the point? Things like "this is" might be a bit excessive. Something like "Profile User Avatar" or "Profile Picture" or somethin like that makes for a easier skim.

Reply:
Ok, thanks for the input!